app/services/rbac/group_seed.rb

```diff
@@ -0,0 +1,19 @@
+module RBAC
+  class GroupSeed
+    $seeded_tenants = {}
+
+    def initialize(user)
+      @user = user
+    end
+
+    def process
+      self.tap do
+        return unless @user.org_admin?
+        return if $seeded_tenants[ActsAsTenant.current_tenant.external_tenant]
+
+        RBAC::Seed.new(Rails.root.join('data', 'rbac_catalog_seed.yml')).process
+        $seeded_tenants[ActsAsTenant.current_tenant.external_tenant] = true
```

Comment:
The idea here was that we would skip this check once the seeding has been done, but when a pod restarts we would validate it again.  Protects against the group being deleted.  Does not protect against it be renamed, but that might be a doc issue for starters.

Reply:
@gmcculloug Is this something that would be needed by Approval too?

Reply:
Yes, definitely.  I thought we would get the pattern worked out here and they apply to the Approval service.